docs/book/glossary.md

```diff
@@ -7,33 +7,36 @@ concepts, their canonical name, and how they relate to or differ from other term
+| 2D encoding                       | encoding scheme used in *RedStuff* where each *blob* is encoded in two dimensions—*primary* and *secondary*—to ensure robust recovery and redundancy.                                       |
+| aggregator                        | service that reconstructs *blobs* by interacting with *SNs* and exposes a simple `HTTP GET` endpoint to *end users*                                                                         |
+| availability period              | the period specified in *storage epochs* for which a *blob* is certified to be available on Walrus                                                                                          |
 | blob                              | single unstructured data object stored on Walrus                                                                                                                                            |
-| permanent blob                    | blob which cannot be deleted by its owner and is guaranteed to be available until at least its expiry epoch (assuming it is valid)                                                          |
-| deletable blob                    | blob which can be deleted by its owner at any time to be able to reuse the storage resource                                                                                                 |
-| shard                             | (disjoint) subset of erasure-encoded data of all *blobs*; at every point in time, a *shard* is assigned to and stored on a single *SN*                                                      |
-| RedStuff                          | our erasure-encoding approach, which uses two different encodings (*primary* and *secondary*) to enable shard recovery; details are available in the [whitepaper](./walrus.pdf)             |
-| sliver                            | erasure-encoded data of one *shard* corresponding to a single blob for one of the two encodings; this contains several erasure-encoded symbols of that blob but not the *blob metadata*     |
-| sliver pair                       | the combination of a shard’s primary and secondary sliver                                                                                                                                   |
 | blob ID                           | cryptographic ID computed from a *blob*’s *slivers*                                                                                                                                         |
 | blob metadata                     | metadata of one *blob*; in particular, this contains a hash per *shard* to enable the authentication of *slivers* and recovery symbols                                                      |
+| (blob) reconstruction             | decoding of the primary *slivers* to obtain the blob; includes re-encoding the *blob* and checking the Merkle proofs                                                                        |
```

Comment:
Suggestion: Change this to `reconstruction (of a blob)` and sort this under `r`.